doc/classes/HTTPRequest.xml

```diff
@@ -203,7 +204,7 @@
 			<param index="3" name="request_data_raw" type="PackedByteArray" default="PackedByteArray()" />
 			<description>
 				Creates request on the underlying [HTTPClient] using a raw array of bytes for the request body. If there is no configuration errors, it tries to connect using [method HTTPClient.connect_to_host] and passes parameters onto [method HTTPClient.request].
-				Returns [constant OK] if request is successfully created. (Does not imply that the server has responded), [constant ERR_UNCONFIGURED] if not in the tree, [constant ERR_BUSY] if still processing previous request, [constant ERR_INVALID_PARAMETER] if given string is not a valid URL format, or [constant ERR_CANT_CONNECT] if not using thread and the [HTTPClient] cannot connect to host.
+				Returns [constant OK] if request is successfully created. (Does not imply that the server has responded), [constant ERR_UNCONFIGURED] if not in the tree, [constant ERR_BUSY] if still processing previous request, [constant ERR_INVALID_PARAMETER] if given string is not a valid URL format or if the method is [HTTPClient.METHOD_CONNECT] (see [request]), or [constant ERR_CANT_CONNECT] if not using thread and the [HTTPClient] cannot connect to host.
```

Comment:
```suggestion
				Returns [constant OK] if request is successfully created. (Does not imply that the server has responded), [constant ERR_UNCONFIGURED] if not in the tree, [constant ERR_BUSY] if still processing previous request, [constant ERR_INVALID_PARAMETER] if given string is not a valid URL format or if the method is [constant HTTPClient.METHOD_CONNECT] (see [method request]), or [constant ERR_CANT_CONNECT] if not using thread and the [HTTPClient] cannot connect to host.
```

This should fix it I think.

Reply:
Docs job is green. You're awesome! Thanks!